bundles/com.espressif.idf.ui/META-INF/MANIFEST.MF

```diff
@@ -52,5 +52,5 @@ Export-Package: com.espressif.idf.ui,
 Bundle-ClassPath: .,
  lib/commonmark-0.16.1.jar,
  lib/ini4j-0.5.4.jar,
- lib/gson-2.8.7.jar
-
+ lib/gson-2.8.7.jar,
+ lib/commons-compress-1.21.jar
```

Comment:
Could you check if the jar is signed? macOS app notarization won't accept jars which are not signed

Reply:
Hi @kolipakakondal this is not signed can you please point me in the right direction to see how to sign this

Reply:
I have shared the details over a mail